package.json

```diff
@@ -28,6 +28,9 @@
     "deploy:docs": "gh-pages -d docs/build -m 'deploy docs [ci skip]'",
     "lint": "cross-env NODE_ENV=production eslint .",
     "lint:fix": "yarn lint --fix",
+    "prettier": "prettier --list-different \"**/*.{js,jsx,ts,tsx}\"",
+    "prettier:fix": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
```

Comment:
Did you test it under Windows? In another project I had trouble with glob patterns as they tried to match files from `C:\` and not from the current directory. The fix was to prefix the glob pattern with `./`.